project/src/main/webapp/upload-page/upload.html

```diff
@@ -8,9 +8,9 @@
   </head>
   <body>
       <div id="content"> 
-        <form action="index.html">
-            <input type="submit" value="Return to Main Page" />
-        </form>
+        <a href="https://summer22-sps-22.uc.r.appspot.com/">
```

Comment:
I think the href can be "/", relative to your site's homepage. Using relative paths to refer to other pages in your site is usually a good practice because if your website changes its URL, it won't break.

Reply:
fixed